script/version.js

```diff
@@ -1,5 +1,5 @@
 define('bigscreenplayer/version',
   function () {
-    return '3.8.0';
```

Comment:
Looks like you haven't correctly run ``npm run pre-publish:patch` here, otherwise the `package.json` and package-lock.json` would also be updated.